src/core/js/helpers.js

```diff
@@ -101,11 +101,38 @@ define([
 
         /**
          * makes the _globals object in course.json available to a template
-         */ 
+         */
         import_globals: function(context) {
-            if(!context.data.root._globals) {
-                context.data.root._globals = Adapt.course.get('_globals');
+            if (context.data.root._globals) return "";
+            context.data.root._globals = Adapt.course.get('_globals');
+            return "";
+        },
+
+        /**
+         * makes the course data available to a template
+         */
+        import_data: function(context) {
+            
+            var directImport = ['config','course'];
+            for (var i = 0, l = directImport.length; i < l; i++) {
+                var name = directImport[i];
+                if (context.data.root['_'+name]) continue;
+                context.data.root['_'+name] = Adapt[name].toJSON();
+            }
+
+            var indexedImport = ['contentObjects','articles','blocks','components'];
+            for (var i = 0, l = indexedImport.length; i < l; i++) {
+                var name = indexedImport[i];
```

Comment:
variable redeclaration, strictly speaking (as you've already declared it on L118 and `var` variables aren't block-scoped)